src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -65,47 +62,77 @@ export const TravelAidScreenComponent = ({
-                  predictionInaccurate: false,
-                  serviceJourney: serviceJourney ?? {},
-                  destinationDisplay:
-                    state.focusedEstimatedCall.destinationDisplay,
-                }}
-              />
-            </GenericSectionItem>
-            <GenericSectionItem>
-              <View style={styles.sectionContainer}>
-                {state.status === TravelAidStatus.NoRealtime && (
-                  <MessageInfoBox
-                    type="error"
-                    title={t(TravelAidTexts.noRealtimeError.title)}
-                    message={t(TravelAidTexts.noRealtimeError.message)}
-                  />
-                )}
-                <View style={styles.subContainer}>
-                  <ThemeText type="body__tertiary--bold">
-                    {getStopHeader(state.status, t)}
-                  </ThemeText>
-                  <ThemeText type="heading__title">
-                    {state.focusedEstimatedCall.quay.stopPlace?.name}{' '}
-                    {state.focusedEstimatedCall.quay.publicCode}
-                  </ThemeText>
-                </View>
-                <TimeInfo state={state} />
-              </View>
-            </GenericSectionItem>
-          </Section>
+        {isLoading && <ActivityIndicator size="large" />}
+        {(isError || (!isLoading && !serviceJourney?.estimatedCalls)) && (
+          <MessageInfoBox
+            type="error"
+            message={t(TravelAidTexts.error.message)}
+            onPressConfig={{action: refetch, text: t(dictionary.retry)}}
+          />
+        )}
+        {serviceJourney?.estimatedCalls && (
+          <TravelAidSection
+            serviceJourney={serviceJourney}
+            fromQuayId={serviceJourneyDeparture.fromQuayId}
+          />
         )}
       </ScrollView>
     </SafeAreaView>
   );
 };
 
+const TravelAidSection = ({
+  serviceJourney,
+  fromQuayId,
+}: {
+  serviceJourney: ServiceJourneyWithEstCallsFragment;
+  fromQuayId?: string;
+}) => {
+  const styles = useStyles();
+  const {t} = useTranslation();
+
+  const {status, focusedEstimatedCall} = getFocusedEstimatedCall(
+    serviceJourney.estimatedCalls!,
+    fromQuayId,
+  );
+
+  return (
+    <Section>
+      <GenericSectionItem style={styles.sectionContainer}>
+        <EstimatedCallInfo
+          departure={{
+            cancellation: false,
+            predictionInaccurate: false,
+            serviceJourney: serviceJourney ?? {},
+            destinationDisplay: focusedEstimatedCall.destinationDisplay,
+          }}
+        />
+      </GenericSectionItem>
+      <GenericSectionItem>
+        <View style={styles.sectionContainer}>
+          {status === TravelAidStatus.NoRealtime && (
+            <MessageInfoBox
+              type="error"
+              title={t(TravelAidTexts.noRealtimeError.title)}
+              message={t(TravelAidTexts.noRealtimeError.message)}
+            />
+          )}
```

Comment:
It is kinda a shift in thought that not having real time data is such a bad thing that it gives a red error message. We generally have treated not having real time data just as normal data, presenting the aimed times instead. 

Seb has also been suggesting some changes that uses and shows realtime data even less than we do now, which would be a change in the other direction than this. I think it might be nice to have a general discussion with Seb and design lead on how central real time data should be in the app going forward, and whether not having realtime data is considered a faulty state or not.